storage_service/templates/locations/pipeline_form.html

```diff
@@ -1,6 +1,7 @@
 {% extends "base.html" %}
+{% load i18n %}
 
-{% block page_title %}{{action}} Pipeline{% endblock %}
+{% block page_title %}{% blocktrans %}{{action}} Pipeline{% endblocktrans %}{% endblock %}
```

Comment:
I ran into a similar case while I was working on the archivematica-fpr-admin project and I wasn't totally sure if every language is going to be fully covered with just one translation for all the permutations of "{{ action }} Pipeline" - perhaps would be more convenient for translators to have multiple strings, e.g. "Edit pipeline", "Open pipeline", "Delete pipeline". I'm not suggesting you to change it but I figured I'd raise the point for further discussion.